app/models/podcast.rb

```diff
@@ -14,10 +14,22 @@ class Podcast < BaseModel
 
   acts_as_paranoid
 
-  scope :published, -> { where('published_at IS NOT NULL') }
+  scope :published, -> { where('published_at IS NOT NULL AND published_at <= now()') }
 
-  after_save do
-    DateUpdater.last_build_date(self)
+  def pub_date
+    if max_episode_published_at
+      [published_at, max_episode_published_at].max
```

Comment:
I suppose we could.

I'm trying to think if we lose anything that way - I guess if you wanted to launch a podcast in the future, you would have to do so by having the first episode with a future published_at

I had set it this way thinking you might have a different published_at for the podcast itself for some reason, but yeah, pretty much dependent on the episodes.